src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -68,21 +68,20 @@ class WorkspaceTabs extends PureComponent {
           // some pages highlight a tab even when they're not on that url
           onClick: href === window.location.hash ? refresh : undefined,
           href
-        }, tabName),
-        navSeparator
+        }, tabName)
       ])
     }
     const isOwner = workspace && Utils.isOwner(workspace.accessLevel)
     const menuIcon = iconName => {
       return icon(iconName, { size: 15, style: { marginRight: '.5rem' } })
     }
     return contextBar({ style: styles.tabContainer }, [
-      navSeparator,
-      navTab({ tabName: 'dashboard', href: Nav.getLink('workspace', { namespace, name }) }),
-      navTab({ tabName: 'notebooks', href: Nav.getLink('workspace-notebooks', { namespace, name }) }),
-      navTab({ tabName: 'data', href: Nav.getLink('workspace-data', { namespace, name }) }),
-      navTab({ tabName: 'tools', href: Nav.getLink('workspace-tools', { namespace, name }) }),
-      navTab({ tabName: 'job history', href: Nav.getLink('workspace-job-history', { namespace, name }) }),
+      activeTab === 'dashboard' ? undefined : navSeparator,
```

Comment:
The purpose of these changes is so that the dividers between the workspace Tabs only show up when they are not adjacent to an active tab. The reason is that the tab dividers are the same color as the active tab, which when put together made it look like Saturn has trouble coloring in the lines.
<img width="1105" alt="screen shot 2018-10-05 at 12 31 03 pm" src="https://user-images.githubusercontent.com/2395211/46547642-819f0000-c89a-11e8-964a-93f659493887.png">
If you have any ideas for a more elegant way to get the safe effect, let me know.